src/main/java/org/javawebstack/orm/query/QueryColumn.java

```diff
@@ -37,4 +43,9 @@ public String toString(TableInfo info) {
         return Arrays.stream((info != null ? info.getColumnName(name) : name).split("\\.")).map(s -> "`" + s + "`").collect(Collectors.joining("."));
     }
 
+    private static void validateName(String name) {
+        if(!NAME_PATTERN.matcher(name).matches())
+            throw new ORMQueryException("Invalid column name '" + name + "' (Use raw in case you know what you're doing)");
```

Comment:
This solves the other aspect of the issue of #14. If this branch is merged the issue can be closed.